features/admin.application-roles.v1/components/application-role-groups.tsx

```diff
@@ -395,16 +408,18 @@ const ApplicationRoleGroups = (props: ApplicationRoleGroupsProps): ReactElement
                                             fluid
                                         />
                                     </Grid.Column>
-                                    <Grid.Column width={ 10 }>
-                                        <PrimaryButton
-                                            data-testid={ `${ componentId }-groups-list-assign-button` }
-                                            onClick={ () => setShowWizard(true) }
-                                            floated="right"
-                                        >
-                                            <Icon name="add"/>
-                                            { t("extensions:console.applicationRoles.roleGroups.assignGroup") }
-                                        </PrimaryButton>
-                                    </Grid.Column>
+                                    <Show when={ featureConfig?.applicationRoles?.scopes?.update }>
+                                        <Grid.Column width={ 10 }>
+                                            <PrimaryButton
+                                                data-testid={ `${ componentId }-groups-list-assign-button` }
+                                                onClick={ () => setShowWizard(true) }
+                                                floated="right"
+                                            >
+                                                <Icon name="add"/>
```

Comment:
Are we using this for consistency? otherwise we can use <PlusIcon />

Reply:
This was an existing icon in the component. I just provided RBAC support with this PR.
Anyway application roles will be deprecated soon. Hence, no need to invest on a refactor :)